pack_and_terraform_aws.bash

```diff
@@ -188,18 +188,6 @@ if echo "$runtf" | grep -iq "^y"; then
         [ -z "$aws_s3_bucket" ] && aws_s3_bucket="cdis-terraform-states"
     fi
 
-    if [ -z "$base_ami" ]; then
```

Comment:
why are these removed?

Reply:
terraform has switched over to copying the public ubuntu16 images - those variables are no longer present in the .tf files